app/models/broadcast.js

```diff
@@ -1,41 +0,0 @@
-import Ember from 'ember';
```

Comment:
I wasn't able to find any of the methods (e.g. `isSeen` or `setSeen`) being used anywhere else in the app. The rest of the app appeared to use the storage service. I will double-check though.